### PR TITLE
[p5.js - es- Cristian] Shape, Half 3D and Half Constants

### PR DESCRIPTION
**Pull request by holomorfo**
# Shape, half 3D and Half Constants

CC: @Guirdo 

This PR includes the following translations:
- All in `Shapes`
- First half of `3D` until `copyToContext`
- Constants from `HALF_PI` to `TAU

### CODE REVIEW

**Comment by holomorfo** [comment]
Hi @Qianqianye and @limzykenneth  I have a question, on the mdx files do we need to translate the `module` and `submodule`
For example:
```
module: Shape
submodule: Vertex
```
Since I believe these names are coupled to the actual library not sure if we should translate them, can you confirm please?

CC: @Guirdo

**Comment by limzykenneth** [comment]
@holomorfo The `module` and `submodule` fields should not be translated as they are read by the build to put things in the right place.

File: src/content/reference/es/p5/sphere.mdx [hunk 1]
```diff
@@ -4,41 +4,18 @@ module: Shape
...
+description: >
+  <p>Dibuja una esfera.</p>
 
-  <p>The third parameter, <code>detailY</code>, is also optional. If a
-  <code>Number</code> is passed,
+  <p>Una esfera es una forma 3D con caras triangulares que se conectan para formar una superficie redonda. Las esferas con pocas caras parecen cristales. Las esferas con muchas caras tienen superficies suaves y parecen bolas.</p>
 
-  as in <code>sphere(20, 5, 2)</code>, it sets the number of triangle
-  subdivisions to
+  <p>El primer parámetro, <code>radius</code>, es opcional. Si se pasa un <code>Número</code>, como en <code>sphere(20)</code>, establece el radio de la esfera. Por defecto, <code>radius</code> es 50.</p>
```
Comment (Guirdo) [inline src/content/reference/es/p5/sphere.mdx:12]: etiqueta code traducida `<code>Número</code>`

File: src/content/reference/es/p5/box.mdx
Comment (Guirdo) [inline src/content/reference/es/p5/box.mdx:1]: Sugiero traducir "box" por "caja", ya que un cubo es un prisma rectangular pero no todos los prismas rectangulares son cubos

**Review by Guirdo — CHANGES_REQUESTED** [review]
Gracias Cristian! Te dejo algunos observaciones

**Comment by holomorfo** [comment]
Hola @Guirdo ya hice las correcciones necesarias, muchas gracias por encontrar esos detalles (y)

File: src/content/reference/es/p5/cylinder.mdx [hunk 2]
```diff
@@ -4,69 +4,66 @@ module: Shape
...
+  <p>El primer parámetro, <code>radius</code>, es opcional. Si se pasa un
+  <code>Number</code>, como en
 
-  <code>cylinder(20)</code>, it sets the radius of the cylinder’s base. By
-  default,
+  <code>cylinder(20)</code>, establece el radio de la base del cilindro. Por
+  defecto,
 
-  <code>radius</code> is 50.</p>
+  el <code>radio</code> es 50.</p>
```
Comment (Guirdo) [inline src/content/reference/es/p5/cylinder.mdx:21]: `Code` traducido

**Review by Guirdo — CHANGES_REQUESTED** [review]
Solo un último detalle y sería todo

File: src/content/reference/es/p5/cylinder.mdx (hunk 2 above)
Comment (holomorfo) [reply src/content/reference/es/p5/cylinder.mdx]: listo, corregido, muchas gracias ^_^

**Review by Guirdo — APPROVED** [review]
Muchas gracias Cristian! Aprobado :100:

**Comment by Guirdo** [comment]
Hi @Qianqianye @limzykenneth ! This PR is ready, but I'm not sure why it is not passing the tests

**Comment by holomorfo** [comment]
> Hi @Qianqianye @limzykenneth ! This PR is ready, but I'm not sure why it is not passing the tests

I'm seeing in the logs: 
"bad indentation of a mapping entry"

But I'm unable to find a specific line or log to check where this might be happening.
CC: @limzykenneth

**Comment by limzykenneth** [comment]
`p5/bezierVertex.mdx` has indentation problem around the overloads. There may be more. To check, you can start the dev server with `npm run dev` and open up the local site in the browser which should show you where exactly the problem is. On each change you may need to restart the local server for it to pick up the changes though.

**Comment by holomorfo** [comment]
> `p5/bezierVertex.mdx` has indentation problem around the overloads. There may be more. To check, you can start the dev server with `npm run dev` and open up the local site in the browser which should show you where exactly the problem is. On each change you may need to restart the local server for it to pick up the changes though.

@limzykenneth Thanks for the details, I have tried as you suggested, found and solved the issues, the PR is passing now.
@Qianqianye this PR is ready for merging:

CC: @Guirdo

**Comment by Qianqianye** [comment]
Looks great, thanks!

**Pull request merged**